app/grandchallenge/discussion_forums/templates/discussion_forums/forum_base.html

```diff
@@ -0,0 +1,38 @@
+{% extends "base.html" %}
+{% load static %}
+{% load url %}
+{% load guardian_tags %}
+
+
+{% block title %}{% block sub_title %}{% endblock sub_title %} Forum{% endblock title %}
+
+{% block breadcrumbs %}
+    <ol class="breadcrumb">
+        <li class="breadcrumb-item"><a
+                href="{% url 'challenges:list' %}">Challenges</a>
+        </li>
+        <li class="breadcrumb-item"><a
+                href="{{ challenge.get_absolute_url }}">{% firstof challenge.title challenge.short_name %}</a></li>
+        {% block extra_breadcrumbs %}{% endblock %}
+    </ol>
+{% endblock %}
+
+{% block content %}
+    <div class="row mb-3 justify-content-between">
+        <div class="col-8"><h1>Forum</h1></div>
+        {% get_obj_perms request.user for forum as "user_perms" %}
+        {% if 'view_forum' in user_perms %}
+            <div class="col-4">
+                <div class="float-right controls-link-wrapper">
+                    {# TODO: my posts view button #}
+                    {% include 'actstream/partials/follow_unfollow_links.html' with object=forum %}
+                    <a href="{% url 'notifications:follow-list' %}" class="btn btn-dark btn-sm"><i class="fas fa-bookmark">&nbsp;</i>My Subscriptions</a>
```

Comment:
I think there are a few inconsistent uses of non-breaking spaces to separate icons and text. Here there is one within the `<i>` tag, others have one or two outside the `<i>` tag, others do not have any. We should try and be consistent about how these are used.

Reply:
I completely agree. I had started removing the non-breaking spaces, but clearly missed a few.